config/fcos/v1_5_exp/validate.go

```diff
@@ -51,3 +51,14 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (disk Disk) Validate(c path.ContextPath) (r report.Report) {
```

Comment:
While this logic seems to be checking for a partition number == 0 and wipe-table == false
What are we going to do for the special case of coreos-boot-disk?

Reply:
have that added now, anytime the root disk is specified it skips the rest of the test